### PR TITLE
Mantic testing step one

### PR DESCRIPTION
**Pull request by renanrodrigo**
## Why is this needed?
This PR solves all of our problems because we want to build and test on mantic now, and want to include as many lunar tests as possible to the CI.

## Test Steps
Run the modified integration tests

## Checklist
 - [n/a] I have updated or added any unit tests accordingly
 - [x] I have updated or added any integration tests accordingly
 - [x] I have updated or added any documentation accordingly

## Does this PR require extra reviews?
 - [ ] Yes
 - [x] No

no-gh no-lp no-jira

### CODE REVIEW

File: dev-docs/howtoguides/testing.md [hunk 1]
```diff
@@ -71,8 +71,9 @@ tox -e behave-lxd-20.04 features/unattached_commands.feature:28
 ```
 
 As can be seen, this will run behave tests only for release 20.04 (Focal Fossa). We are currently
-supporting 5 distinct releases:
+supporting 6 distinct releases:
 
+* 23.04 (Luanr Lobster)
```
Comment (aciba90) [inline dev-docs/howtoguides/testing.md:76]:
```suggestion
* 23.04 (Lunar Lobster)
```

Comment (renanrodrigo) [reply dev-docs/howtoguides/testing.md:76]: 23.40 (Terrible Typo)

**Review by orndorffgrant — APPROVED** [review]
This seems to be working now? I re-ran the failed CI tests because they looked like flakes.

**Comment by renanrodrigo** [comment]
Yeah it seems the Jammy version of `debootstrap` is fine!
Flakes gone, I'm gonna merge this, and we can iterate over bad tests in the future.

**Pull request merged**